fragments/csi/README.md

```diff
@@ -0,0 +1,3 @@
+# csi plugins
+
+This is a stub for CSI plugins which are coming in the future. We are currently working on a specification for including them as part of the solution. 
```

Comment:
What is the point of making this announcement in the code? I do not hink the code is the right place for that.